.travis.yml

```diff
@@ -1,24 +1,33 @@
 language: python
-matrix:
-  include:
-    - python: 2.7
-      env: SYMPY_VERSION="oldest"
-    - python: 2.7
-      env: SYMPY_VERSION="latest"
-    - python: 2.7
-      env: SYMPY_VERSION="master"
-virtualenv:
-  system_site_packages: true
```

Comment:
So travis creates a default virtualenv for Python that doesn't have access to the system python packages. And it also installs a few other packages into this venv (numpy, mock, nose, etc). The process I had here was to first remove the venv numpy because it was a newer version of numpy, and then the venv python will use the system numpy, scipy, etc. which are the versions that shipped with Ubuntu 12.04. If the `system_site_packages` flag isn't set, then I don't think that is what happens. It may be using conda's packages or something in this case.